shared/profile/wot-author/container.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react'
+import {Question1, Question2} from '.'
+import * as Container from '../../util/container'
+import * as RouteTreeGen from '../../actions/route-tree-gen'
+
+export type Props = Container.RouteProps<{username: string}>
+
+export const WotAuthorRoot = (props: Props) => {
+  const voucheeUsername = Container.getRouteProps(props, 'username', '')
+  const dispatch = Container.useDispatch()
+  const [whichQuestion, setWhichQuestion] = React.useState<'question1' | 'question2'>('question1')
+  let {error, initialVerificationType} = Container.useSelector(state => state.profile.wotAuthor)
+  if (!error && !voucheeUsername) {
+    error = 'Routing missing username.'
+  }
+  if (whichQuestion === 'question1') {
+    const proofs = [{key: 'web', value: 'fake.proof'}] // PICNIC-1088 TODO
+    const onSubmit = answer => {
+      console.log(answer) // PICNIC-1087 TODO use values
+      setWhichQuestion('question2')
```

Comment:
so one thing that I don't like about this is that if you did it using the navigator, you'd get a free slide animation to the next screen on mobile, and with this it just pops.

Reply:
Probably okay though.